.github/workflows/build-image-stable.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     with:
       # See https://github.com/ublue-os/aurora/issues/82
       #  +  https://github.com/ublue-os/aurora/issues/120
```

Comment:
I know this is currently just a draft, but we can delete the above comments as well when we want to merge this.